utils/make_symlinks.bash

```diff
@@ -0,0 +1,18 @@
+# This will be a problem if these simulations are ever removed from the publication archives!
+for i in $(seq 1 20); do
+    hsi ln -s /home/projects/e3sm/www/publication-archives/pub_archive_E3SM_1_0_LE_historical_ens$i /home/projects/e3sm/www/WaterCycle/E3SMv1/LR/LE_historical_ens$i
+done
+
+for i in $(seq 1 20); do
+    hsi ln -s /home/projects/e3sm/www/publication-archives/pub_archive_E3SM_1_0_LE_ssp370_ens$i /home/projects/e3sm/www/WaterCycle/E3SMv1/LR/LE_ssp370_ens$i
+done
+
+# Symlink last remaining large simulation
+# This will be a problem if ndk ever deletes the source!
```

Comment:
@chengzhuzhang I meant to include this in the self-review I just posted. The symlinks are fine as long as we are guaranteed that people don't delete the source directories like `/home/projects/e3sm/www/publication-archives/` or `/home/n/ndk/2019/theta.20190910.branch_noCNT.n825def.unc06.A_WCYCL1950S_CMIP6_HR.ne120_oRRS18v3_ICG`. Is that something we can be sure of?

Reply:
I think so, tagging directory owners @TonyB9000 and @ndkeen, please make sure not to delete above directories.